pl_bolts/datamodules/emnist_datamodule.py

```diff
@@ -0,0 +1,240 @@
+
+from pl_bolts.datamodules.vision_datamodule import VisionDataModule
+from pl_bolts.datasets import EMNIST
+from pl_bolts.utils import _TORCHVISION_AVAILABLE
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+if _TORCHVISION_AVAILABLE:
+    from torchvision import transforms as transform_lib
+else:  # pragma: no cover
+    warn_missing_pkg('torchvision')
+
+
+class EMNISTDataModule(VisionDataModule):
+    """
+    .. figure:: https://user-images.githubusercontent.com/4632336/123210742-4d6b3380-d477-11eb-80da-3e9a74a18a07.png
+        :width: 400
+        :alt: EMNIST
+
+    .. list-table:: Dataset information (source: `EMNIST: an extension of MNIST to handwritten
+        letters <https://arxiv.org/abs/1702.05373>`_ [Table-II])
+       :header-rows: 1
+
+       * - Split Name
+         - No. classes
+         - Train set size
+         - Test set size
+         - Validation set
+         - Total size
+       * - ``"byclass"``
+         - 62
+         - 697,932
+         - 116,323
+         - No
+         - 814,255
+       * - ``byclass``
+         - 62
+         - 697,932
+         - 116,323
+         - No
+         - 814,255
+       * - ``bymerge``
+         - 47
+         - 697,932
+         - 116,323
+         - No
+         - 814,255
+       * - ``balanced``
+         - 47
+         - 112,800
+         - 18,800
+         - Yes
+         - 131,600
+       * - ``digits``
+         - 10
+         - 240,000
+         - 40,000
+         - Yes
+         - 280,000
+       * - ``letters``
+         - 37
+         - 88,800
+         - 14,800
+         - Yes
+         - 103,600
+       * - ``mnist``
+         - 10
+         - 60,000
+         - 10,000
+         - Yes
+         - 70,000
+
+    |
+
+    Standard EMNIST, train, val, test-splits and transforms
+
+    Transforms::
+
+        emnist_transforms = transform_lib.Compose([
+            transform_lib.ToTensor()
+        ])
+
+    Example::
+
+        from pl_bolts.datamodules import EMNISTDataModule
+        dm = EMNISTDataModule('.')
+        model = LitModel()
+        Trainer().fit(model, datamodule=dm)
+    """
+    name = "emnist"
+    dataset_cls = EMNIST
+    dims = (1, 28, 28)
+    # _DEFAULT_NO_VALIDATION_VAL_SPLIT: This is the `val_split` to use when
+    # "validation = False" for a given split in the metadata and the user-input
+    # for `val_split` is `None`.
+    _DEFAULT_NO_VALIDATION_VAL_SPLIT: Union[int, float] = 0
+
+    def __init__(
+        self,
+        data_dir: Optional[str] = None,
+        split: str = 'mnist',
+        val_split: Optional[Union[int, float]] = 0.2,
+        num_workers: int = 16,
+        normalize: bool = False,
+        batch_size: int = 32,
+        seed: int = 42,
+        shuffle: bool = False,
+        pin_memory: bool = False,
+        drop_last: bool = False,
+        *args: Any,
+        **kwargs: Any,
+    ) -> None:
+        """
+        Args:
+            data_dir (string, optional): Where to save/load the data.
```

Comment:
Could you remove typing in the docs to make it consistent with other docstrings? Types are automatically rendered in the docs :]
```suggestion
            data_dir: Where to save/load the data.
```

Reply:
@akihironitta sure :D